app/assets/javascripts/events.js

```diff
@@ -0,0 +1,5 @@
+jQuery(document).ready(function($) {
+    $(".clickable-row").click(function() {
+        window.location = $(this).data("href");
```

Comment:
Turns out that you can have an `a` element between `tbody` and `tr`. HTML won't allow it.
So, kept this in and made an invisible `a` around the name for testing.